parcellab/common/templates/_keda.tpl

```diff
@@ -36,6 +36,7 @@ spec:
         queueLength: "{{ $service.keda.queueLength }}"
         queueURL: {{ $service.keda.queueURL }}
         identityOwner: pod
+        scaleOnInFlight: {{ $service.keda.scaleOnInFlight }}
```

Comment:
should this have a default value?

Reply:
yes, totally  - @luisuribe please add a default value when $service.keda.scaleOnInFlight is not set and also add the value to the values.yaml respectively to ensure users of the chart are aware of this setting

Reply:
Done!